api/mysql/database.sql

```diff
@@ -73,7 +73,8 @@ CREATE TABLE `WorksheetCourses` (
   `netId` char(8) NOT NULL,
   `crn` mediumint(8) UNSIGNED NOT NULL,
   `season` mediumint(8) UNSIGNED NOT NULL,
-  `worksheetNumber` mediumint(8) UNSIGNED DEFAULT 0
+  `worksheetNumber` mediumint(8) UNSIGNED DEFAULT 0 COMMENT 'Multiple worksheets!',
```

Comment:
```suggestion
  `worksheetNumber` mediumint(8) UNSIGNED DEFAULT 0 COMMENT 'Indicator of which of the user's worksheets this belongs to',
```

Reply:
@bearsyankees This is the same comment as in the prod DB :P

Reply:
ya and i made the prod one but i think this is more descriptive lol i was lazy